reddit_liveupdate/pages.py

```diff
@@ -170,6 +171,24 @@ def __init__(self, **kwargs):
 
 
 class LiveUpdateEventFocusPage(LiveUpdateEventPage):
+    def __init__(self, focused_update, **kwargs):
+        og_data = {
+            "type": "article",
+            "url": make_event_url(c.liveupdate_event._id),
+            "description": trunc_string(
+                focused_update.body.strip(), MAX_DESCRIPTION_LENGTH),
+            "image": static("liveupdate-logo.png"),
+            "image:width": "300",
+            "image:height": "300",
+            "site_name": g.short_description,
```

Comment:
Oh - I think yes.

On Friday, July 15, 2016, Neil Williams <notifications@github.com> wrote:

> In reddit_liveupdate/pages.py
> <https://github.com/reddit/reddit-plugin-liveupdate/pull/137#discussion_r71061614>
> :
>
> > @@ -170,6 +171,24 @@ def __init__(self, **kwargs):
> >
> >
> >  class LiveUpdateEventFocusPage(LiveUpdateEventPage):
> > +    def __init__(self, focused_update, **kwargs):
> > +        og_data = {
> > +            "type": "article",
> > +            "url": make_event_url(c.liveupdate_event._id),
> > +            "description": trunc_string(
> > +                focused_update.body.strip(), MAX_DESCRIPTION_LENGTH),
> > +            "image": static("liveupdate-logo.png"),
> > +            "image:width": "300",
> > +            "image:height": "300",
> > +            "site_name": g.short_description,
>
> This is the same site_name as on the other parts of liveupdate. Should I
> change the other as well?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/reddit/reddit-plugin-liveupdate/pull/137/files/260c5c9e672eb49f153c021e3f08804af4ef2ee8#r71061614>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AARc4G_UreOixoFRLlAeucGvj6PTXGeWks5qWHJPgaJpZM4JN4qe>
> .
>